CONTRIBUTING.md

```diff
@@ -361,87 +133,3 @@ Comments have to be written in **English** everywhere. Please use markdown where
 
 ### Documenting new features
 Please update the changelog with every feature/fix you provide so we can keep track of changes for new versions of JuPedSim.
```

Comment:
Do we keep the changelog up-to-date?

Reply:
I guess we are supposed to but didn't do recently